sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f

```diff
@@ -1134,9 +1134,7 @@ SUBROUTINE CALRAD_WCLOUD
                        id(1:25) = 0
                        id(02) = 133
                        id(8) = 175 + ixchan 
```

Comment:
Is there a reason to keep ID( ) array in the code anymore since this is used for grib1 metadata? I don't think it's needed elsewhere?